var/spack/repos/builtin/packages/ruby/package.py

```diff
@@ -114,3 +114,15 @@ def post_install(self):
                                             'rubygems',
                                             'ssl_certs')
             install(rubygems_updated_cert_path, rubygems_certs_path)
+
+        rbconfig = '{0}/lib/ruby/{1}.0/x86_64-linux/rbconfig.rb'.format(
```

Comment:
Is the `x86_64-linux` directory correct for macOS and aarch64?

Reply:
Unknown. I don't have access to either to check. I'll make this addition conditional